packages/video_player/video_player/lib/video_player.dart

```diff
@@ -757,12 +757,12 @@ class _VideoProgressIndicatorState extends State<VideoProgressIndicator> {
         fit: StackFit.passthrough,
         children: <Widget>[
           LinearProgressIndicator(
-            value: maxBuffering / duration,
+            value: duration > 0 ? maxBuffering / duration : 0,
```

Comment:
Does duration 0 means it's indefinite? if so should we use null for value to show an indefinite progress indicator?

Reply:
It's 0 on iOS. Android returns different values

Reply:
Do we have an indication on Android that it's indefinite? what does the progress bar show for indefinite streams on Android?

Reply:
The exoPlayer does have a isDynamic function on the Window. This is set to true when a Live stream is being played. I believe this would be the best solution. [Link to the docs.](https://exoplayer.dev/doc/reference/com/google/android/exoplayer2/Timeline.html#live-limited). 

I could implement a VideoPlayerController().isDurationIndefinite getter that returns true when ExoPlayer returns true on the isDynamic getter on Android, and when isDurationIndefinite returns true on iOS?